SeaBattleTest/Ships.cs

```diff
@@ -0,0 +1,115 @@
+﻿using SeaBattle;
+using System;
+using System.Collections.Generic;
+using SeaBattle;
+using FluentAssertions;
+using System.Drawing;
+using System.Data.Common;
+using System.Runtime.CompilerServices;
+
+namespace SeaBattleTest
+{
+
+    [TestClass]
+    public class Ships
+    {
+        [TestMethod]
+        public void MoveTo_DontMoves_ifPlaceAlreadyOccupied()
+        {
+            //arrange
+            Sea sea = new Sea(10, 10);
+            List<Point> MoveToCoords = new List<Point>(new Point[] { new Point(5, 5), new Point(5, 6), new Point(5, 7), new Point(5, 8) });
+
+            List<Point> HybrydCoords = new List<Point>(new Point[] { new Point(3, 3), new Point(3, 4), new Point(3, 5), new Point(3, 6) });
+            HybridShip hybrydo = new HybridShip("Kolipso", HybrydCoords);
+            List<Point> WarCoords = new List<Point>(new Point[] { new Point(5, 5), new Point(5, 6), new Point(5, 7), new Point(5, 8)});
+            BattleShip warrior = new BattleShip("Warr", WarCoords);
+            //act
+            sea.AddShip(hybrydo);
+            sea.AddShip(warrior);
+
+            //assert
+            hybrydo.Invoking(hybrydo => hybrydo.MoveTo(MoveToCoords, sea)).Should().Throw<ArgumentOutOfRangeException>();
+        }
+
+        [TestMethod]
+        public void MoveTo_DontMoves_ifSpeedLowerThanDistance()
```

Comment:
Same a bit of weird naming, you can do better

Reply:
Renamed.